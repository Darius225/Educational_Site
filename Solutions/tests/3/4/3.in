50
31893 14141 23055 1189 26993 29780 13057 30572 3462 13077 5377 6369 27656 814 17687 32190 23381 25562 32417 2232 13538 1029 17130 19388 16756 19889 18174 3705 13715 16536 29629 7710 21051 24595 11311 19045 32214 14394 19991 13264 26983 15949 8395 20274 5640 4179 9794 17010 20118 22528 