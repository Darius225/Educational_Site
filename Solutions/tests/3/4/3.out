814 1029 1189 2232 3462 3705 4179 5377 5640 6369 7710 8395 9794 11311 13057 13077 13264 13538 13715 14141 14394 15949 16536 16756 17010 17130 17687 18174 19045 19388 19889 19991 20118 20274 21051 22528 23055 23381 24595 25562 26983 26993 27656 29629 29780 30572 31893 32190 32214 32417 